### PR TITLE
VIH-5435: Disable "continue" button on self test page

### PR DESCRIPTION
**Pull request by RocheL**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-5435
### Change description ###
modified the component to keep the continue button disabled until the test score is retrieved.
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/self-test-journey/pages/test-your-equipment/test-your-equipment.component.ts [hunk 1]
```diff
@@ -303,7 +309,7 @@ export class TestYourEquipmentComponent extends SuitabilityChoicePageBaseCompone
     }
     this.pexipAPI = null;
     this.logger.event('telemetry:serviceweb:any:selftest:complete');
-    this.retrieveSelfTestScore();
+    // this.retrieveSelfTestScore();
```
Comment (npoxon) [inline ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/self-test-journey/pages/test-your-equipment/test-your-equipment.component.ts]: is this meant to be commented out?

Comment (kkodalikk) [reply ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/self-test-journey/pages/test-your-equipment/test-your-equipment.component.ts]: We can delete this

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/SelfTest/TestYourEquipment.feature [hunk 2]
```diff
@@ -22,23 +22,8 @@ Scenario: Test Your Equipment Representative
 	When the Test Your Equipment video has ended
 	And the user clicks the Watch the video again button
 	Then the Test Your Equipment video begins to play
+	When the Test Your Equipment video has ended
```
Comment (npoxon) [inline ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/SelfTest/TestYourEquipment.feature]: You need to add this to the 'Scenario: Test Your Equipment Individual' test as well

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/self-test-journey/pages/test-your-equipment/test-your-equipment.component.ts (hunk 1 above)
Comment (kkodalikk) [reply ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/self-test-journey/pages/test-your-equipment/test-your-equipment.component.ts]: Done

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/SelfTest/TestYourEquipment.feature (hunk 2 above)
Comment (kkodalikk) [reply ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/SelfTest/TestYourEquipment.feature]: Fixed this.

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/self-test-journey/pages/test-your-equipment/test-your-equipment.component.ts (hunk 1 above)
Comment (npoxon) [reply ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/self-test-journey/pages/test-your-equipment/test-your-equipment.component.ts]: Hey @kkodalikk its still there ^^

**Pull request merged**